path/fount.sh

```
@@ -945,25 +953,50 @@ fount_git_backup_uncommitted() {
-	if command -v git &>/dev/null; then
-		fount_git_backup_uncommitted
-		git -C "$FOUNT_DIR" config core.autocrlf false
-		git -C "$FOUNT_DIR" clean -fd
-		git -C "$FOUNT_DIR" reset --hard "origin/master"
-		git -C "$FOUNT_DIR" gc --aggressive --prune=now --force
+	command -v git &>/dev/null || return 0
+	invoke_git_for_fount config core.autocrlf false
+	local has_head=0 fetch_ok=0
+	if fount_git_ref_exists HEAD; then has_head=1; fi
+	invoke_git_for_fount fetch origin
+	if [ $? -eq 0 ]; then fetch_ok=1; fi
```

Shellcheck (0.11.0)

[style] Check exit code directly with e.g. 'if mycmd;', not indirectly with $?.

(SC2181)